src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Rules/NuGetRestore.xaml

```diff
@@ -20,6 +20,10 @@
   <StringProperty Name="CentralPackageVersionOverrideEnabled"
                   ReadOnly="True"
                   Visible="False" />
+                  
+  <StringProperty Name="CentralPackageFloatingVersionsEnabled"
```

Comment:
Should this be a bool property?

Reply:
I'm not sure, a bunch of these other properties are `true/false` only and are string properties.  Is it better to declare them as booleans?  I can send a follow-up PR to address that if needed.